### DISCUSSION TITLE
Stop pending request on unsubscribe

### DISCUSSION THREAD

**Discussion by JanKolk** [the-guild-org/apollo-angular#1552] [General]
We are currently working with apollo-angular version 2.0.3 and @apollo/client version 3.1.3 but we are not able to cancel the pending requests when calling unsubscribe on the observable. 

Is there a way to cancel the requests when unsubscribe is called. 

Current behaviour.
The unsubscribe finishes but the call continues

Requested behaviour
The unsubscribes also cancels the request to the server.

**Comment by PowerKiKi** [comment]
After a quick glance at the code below, I *think* it should be cancellable. Do you have a reproduction case that demonstrate your issue ?

https://github.com/kamilkisiela/apollo-angular/blob/40597625a51a2de63bb1470389f0302793b3ebc5/packages/apollo-angular/http/src/http-link.ts#L97-L101,
